web/client/components/data/identify/DefaultViewer.jsx

```diff
@@ -192,7 +218,7 @@ class DefaultViewer extends React.Component {
         componentOrder = this.props.isMobile ? componentOrder : reverse(componentOrder);
         return (
             <div className="mapstore-identify-viewer">
-                {!emptyResponses ? componentOrder.map((c)=> c) : this.renderEmptyPages()}
+                {!emptyResponses ? componentOrder.map((c)=> c) : this.renderEmptyPages(this.props)}
```

Comment:
I don't think this is needed, renderEmptyPages takes no arguments
```suggestion
                {!emptyResponses ? componentOrder.map((c)=> c) : this.renderEmptyPages()}
```

Reply:
good catch